rules/php80/tests/Rector/If_/NullsafeOperatorRector/Fixture/fixture_no_return_last.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Rector\Php80\Tests\Rector\If_\NullsafeOperatorRector\Fixture;
+
+class SomeClassNoReturnLast
+{
+    public function f($o)
+    {
+        $o2 = $o->mayFail1();
+        if ($o2 === null) {
+            return null;
+        }
+
+        $o2->mayFail2();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php80\Tests\Rector\If_\NullsafeOperatorRector\Fixture;
+
+class SomeClassNoReturnLast
+{
+    public function f($o)
+    {
+        $o?->mayFail1()?->mayFail2();
```

Comment:
Same here.

Reply:
That's needed to check if that's null, return early, else that uses re-assign with new object and call other method by new new object, ref https://3v4l.org/U9vkk